contracts/MerkleAirdrop.sol

```diff
@@ -23,6 +23,7 @@ contract MerkleAirdrop {
 
     address owner;
     bytes32 public merkleRoot;
+    bool cancelable;
```

Comment:
it's worth to mark it as `public` and define corresponding function info in the smartz constructor